### DISCUSSION TITLE
Issue running brainvisa on Windows via wsl

### DISCUSSION THREAD

**Discussion by jtrossiter** [brainvisa/brainvisa.github.io#173] [Q&A]
I'm having issues using brainvisa (and anatomist) after previously using it no problem, I'm not very experienced in this stuff so apologies if the fix is obvious. I followed the instructions on [this webpage ](https://brainvisa.info/web-5.1/download.html) to install brainvisa with singularity on windows. Whenever I needed to use it previously I would first run xlaunch and then run brainvisa using the ubuntu command prompt. However it's recently stopped working despite me not changing anything. I get the following error message:

xauth:  file /home/jordanr/.Xauthority does not exist
No matches found, authority file "/tmp/casa-distro-z2wg4a5s.Xauthority" not written
Could not use Matplotlib: <class 'ImportError'> Cannot load backend 'Qt5Agg' which requires the 'qt' interactive framework, as 'headless' is currently running
qt.qpa.xcb: could not connect to display 10.255.255.254:0
qt.qpa.plugin: Could not load the Qt platform plugin "xcb" in "" even though it was found.
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Available platform plugins are: eglfs, linuxfb, minimal, minimalegl, offscreen, vnc, xcb.

Aborted

The first two lines of the message are nothing new, they would always appear but brainvisa would still run anyway. What might be the issue here? I've tried some fixes I've seen online, mostly installing different packages, but none have worked

**Comment by sapetnioc** [comment]
I often use Windows with WSL2 without problem. However, I have not used container for months since I am investigating a new Conda package based building system. I recently noticed some compatibility problems with Singularity on a Linux system. Therefore I suspect Singularity to be the source of your problem. You should try to use Apptainer instead of Singularity and see if the problem is fixed. FYI, Apptainer is a fork of Singularity due to a change in the open source model of the project. BrainVISA switched to Apptainer months (years?) ago but we did not update the doc. If the two projects are not compatible anymore, we will should change it quickly.

**Reply by jtrossiter** [reply]
So to be clear, I should reinstall brainvisa following the steps in the doc but using apptainer instead of singularity? And if so, how might I uninstall the current version?

**Comment by denisri** [comment]
No you don't need to reinstall brainvisa, just install apptainer instead of singularity, so that the command "apptainer" is in the path.

**Reply by jtrossiter** [reply]
Hmm I'm still getting the same error mentioned in the original post, I removed singularity, installed apptainer, then ran `apptainer run -c -B $HOME/brainvisa-5.1.1:/casa/setup $HOME/brainvisa-5.1.1/brainvisa-5.1.1.sif`. From there running brainvisa-5.1.1/bin/brainvisa (as I usually would to launch brainvisa) gives me the error
